File: practices/old_enough.py
# MA 6th old enough  

age = int(input("What is your age?"))

if age >= 18:
    print("You are old enough to vote!")
elif age >= 16:
    print("You are old enough to drive!")
elif age == 15:
    print("You are old enough to get a learners permit!")
else:
    if age >= 4:
        print("You are old enough to go to school!")

File: c_programming/projects/old_enough.c
//MA 6th old enough project for c

#include<stdio.h>
#include<strings.h>
int main(void){

int age;
    printf("What is your age?: ");
    scanf("%d", &age);

    if (age >= 18){
        printf("You are old enough to vote!\n");
    }
    else if (age >= 16){
        printf("You are old enough to drive!\n");
    }
    else if (age == 15){
        printf("You are old enough to get a learners permit!\n");
    }
    else if (age >= 4){
        printf("You are old enough to go to school!\n");
    }
    else{
        printf("You are not old enough to go to school!\n");
    }
return 0;
}

